roles/datadog-builder/tasks/datadog-builder.yml

```diff
@@ -23,9 +23,18 @@
 - name: create datadog-builder user
   user:
     name: datadog-builder
+    group: datadog-builder
     home: /var/lib/datadog-builder
     state: present
 
+- name: Create log file
+  file:
+    path: /var/log/datadog-builder.log
+    state: present
+    mode: 0755
+    owner: datadog-builder
+    group: datadog-builder
```

Comment:
I'm not sure if this is a nit or not. 

Normally the way to do this is create a folder in /var/log that has permissions that the application has rights that it can create new files in. This means it shouldn't get into trouble when we do things like logrotation or whatever.

Probably this would be ok, but i think the folder would be better.